packages/db/src/db.ts

```diff
@@ -0,0 +1,27 @@
+import { sharedEnvs } from '@answeroverflow/env/shared';
+
+import {
+	drizzle as psDrizzle,
+	PlanetScaleDatabase,
+} from 'drizzle-orm/planetscale-serverless';
+import { connect } from '@planetscale/database';
+import { fetch } from 'undici';
+
+import { drizzle as mysqlDrizzle } from 'drizzle-orm/mysql2';
+import mysql from 'mysql2/promise';
+
+import * as schema from './schema';
+const connection = connect({
+	url: sharedEnvs.DATABASE_URL,
+	fetch,
+});
+
+export const db: PlanetScaleDatabase<typeof schema> =
```

Comment:
Add a comment about why we're doing this like this, it's so we can connect to non planetscale databases